src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -64,10 +78,18 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
         </p>
       );
       break;
+
+    // Note: This is the template that is used for the Auditorium
     case VenueTemplate.audience:
-      template = <Audience />;
+      template = (
+        <Switch>
+          <Route path={`${match.path}/reactions`} component={ReactionPage} />
```

Comment:
What's the purpose of this screen? I've never heard of this screen being used by anyone and that's why I removed it in my PR.

Reply:
You removed the routes to it, the components still existed.

This displays all of the reactions/shoutouts/etc that have been shared in the respective venue, in a way that allows 'the band' or equivalent to easily see them on a screen, not just few the couple of seconds that they flash up within the venue itself.

Reply:
But I guess since no one navigates to `/reactions`( or does anyone?), this ReactionPage component is never rendered

Reply:
This used to be /band, but I renamed it in this PR when I restored it. No one navigates to it currently because they can’t. This is the same kind of feature as how the banner admin page works.

There is zero harm in having it restored, and given it’s a feature that is done/exists/etc already, it feels pointless to spend additional time questioning/pondering/etc over it.